lib/core/presentation/widgets/event/rsvp_event_success_popup_widget.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class RSVPEventSuccessPopupWidget extends StatelessWidget {
+  const RSVPEventSuccessPopupWidget({
+    super.key,
+    required this.event,
+  });
+
+  final Event event;
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    final t = Translations.of(context);
+    return Scaffold(
+      backgroundColor: colorScheme.primary,
+      body: SafeArea(
+        child: Stack(
+          children: [
+            Column(
+              mainAxisSize: MainAxisSize.max,
+              children: [
+                const _SuccessCircle(),
+                SizedBox(
+                  height: 56.w,
+                ),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: Text(
+                    t.event.youreIn,
+                    style: Typo.extraLarge.copyWith(
+                      fontFamily: FontFamily.nohemiVariable,
+                      fontWeight: FontWeight.w900,
+                      height: 1.2,
+                    ),
+                  ),
+                ),
+                SizedBox(height: Spacing.superExtraSmall),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: Text(
+                    t.event.rsvpSuccessful(eventName: event.title ?? ''),
+                    style: Typo.mediumPlus.copyWith(
+                      fontWeight: FontWeight.w400,
+                      color: colorScheme.onSecondary,
+                    ),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                SizedBox(height: Spacing.xLarge),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: LinearGradientButton(
+                    mode: GradientButtonMode.lavenderMode,
+                    height: Sizing.large,
+                    textStyle: Typo.medium.copyWith(
+                      fontWeight: FontWeight.w600,
+                      color: colorScheme.onPrimary.withOpacity(0.87),
+                      fontFamily: FontFamily.nohemiVariable,
+                    ),
+                    radius: BorderRadius.circular(LemonRadius.small * 2),
+                    label: t.event.takeMeToEvent,
+                  ),
+                )
+              ],
+            ),
+            Positioned(
+              top: 0,
+              left: Spacing.smMedium,
```

Comment:
Better put these 2 Positioned Widgets into a Row to make sure it's aligned with each other

Reply:
👍